e2e/celestia-app/config.toml

```diff
@@ -383,21 +383,21 @@ version = "v0"
 wal_file = "data/cs.wal/wal"
 
 # How long we wait for a proposal block before prevoting nil
-timeout_propose = "50s"
+timeout_propose = "500ms"
```

Comment:
These look like substantial changes to the parameters. What's the reason for that?

Reply:
Just to speed up block production to the maximum without having multiple rounds of consensus.
The reason is having to wait for 600+ blocks to change the window and wait for attestations to be produced, and that would take time to accomplish.